app/search.py

```diff
@@ -51,7 +51,8 @@ def __init__(self, request, topic):
 
         if not request.args.get('lang'):
             # if no lang is provided, the best match of the header accept-language is being used
-            self.lang = request.accept_languages.best_match(SUPPORTED_LANGUAGES)
+            best_lang_match = request.accept_languages.best_match(SUPPORTED_LANGUAGES)
+            self.lang = best_lang_match if best_lang_match else 'de'
         else:
             self.lang = request.args.get('lang')
```

Comment:
It might be clearer as follow
```python
self.lang = 'de'
if 'lang' in request.args:
    self.lang = request.args['lang']
else:
    lang = request.accept_languages.best_match(self.availableLangs)
    if lang: 
       self.lang = lang
```

See https://github.com/geoadmin/service-search-wsgi/blob/7bee0214f1b1dccb4b731c430f33ed1389c55ca5/app/helpers/validation_search.py#L33 so this variable could be set to `SUPPORTED_LANGUAGES` as well.

Might be also a good idea to have the `self.lang` like `self.searchLang` as property with a validation like in https://github.com/geoadmin/service-search-wsgi/blob/7bee0214f1b1dccb4b731c430f33ed1389c55ca5/app/helpers/validation_search.py#L256